### ISSUE TITLE
Support changing default node color

### ISSUE THREAD

**Issue by fedarko** [biocore/empress#143]
Just a small thing that would be nice in the future. For example, drawing nodes black (rather than with the default gray color currently used) increases the contrast significantly:

![image](https://user-images.githubusercontent.com/4177727/77603688-f8381f00-6ecd-11ea-8a94-80d0fcdbdff3.png)

![image](https://user-images.githubusercontent.com/4177727/77603699-fd956980-6ecd-11ea-88bb-0907410f18c0.png)

This could be useful for e.g. vision-impaired folks.

Might be a nice TODO at some point down the line after #137?

**Comment by ElDeveloper** [comment]
Whatever default we select, we should make both background color and tree color user-selectable.

**Comment by fedarko** [comment]
Agreed, having both selectable will be really nice. Just for reference, here's what a "dark mode" tree looks like in Empress ;) -- I imagine sticking with the black/gray on white default will be preferred by most users, but supporting this stuff (and #137) will make it really easy to style Emperor and Empress similarly.

![image](https://user-images.githubusercontent.com/4177727/77683993-848e2480-6f56-11ea-8fb7-63ceda513560.png)

**Comment by ElDeveloper** [comment]
Related to how we want to represent non-unique and non-represented features.

**Comment by fedarko** [comment]
One complication with this issue is that, if we change the default node color to black (which I think would be best IMO), then the root node's special black marker no longer stands out:

![image](https://user-images.githubusercontent.com/4177727/85630666-e713c100-b628-11ea-8f1b-725b851060cb.png)

I think the way around this is changing the root's marker to be different in some other way (e.g. a square / star / unfilled circle / etc.), and maybe making the color of the root configurable as well.

**Comment by ElDeveloper** [comment]
A different shape should work well :+1:

**Comment by fedarko** [comment]
Did a bit more looking into this; it looks like Emperor uses [Spectrum](https://bgrins.github.io/spectrum/) for letting users select colors, so I think for the sake of simplicity it would be good to just use that for this.

(I'd thought it might be possible for us to re-use the vendor JS code between Emperor and Empress in the tandem plots -- letting us avoid duplicating the files unnecessarily -- but it doesn't look like that is currently possible, and I suspect that would probably be more trouble than it's worth.)
